packages/form-js-viewer/src/features/expression-language/FeelersTemplating.js

```diff
@@ -29,6 +65,37 @@ export default class FeelersTemplating {
 
     return evaluateFeelers(template, context, { debug, strict, buildDebugString });
   }
+
+
+  _extractExpressionsWithDepth(template) {
+
+    // build simplified feelers syntax tree
+    const parseTree = feelersParser.parse(template);
```

Comment:
Just for my understanding, the `feelersParser` can also parse `= foo` like expressions?